### ISSUE TITLE
Use Case Example needed for combining MySQLOperator and PythonOperator

### ISSUE THREAD

**Issue by alybel** [apache/airflow#297]
I want to perform a periodic check in a mysql database (connector already built and tested), MySQL operator runs well. My goal is to check if the latest update in the database is older than X, if yes, count as an error and send an Email to the responsible people. I trust it's super easy with Airflow, but I have the following missing links: 
a) If I use the MySQL Operator, how can I store the result of SQL in a variable that is evaluated by a downstream PythonOperator
b) Would it be the proper way to raise an exception in the downstream PythonOperator, count the run as fail and use the email_on_failure option to inform the responsible people? 
c) Would this be the right way to go anyways? Alternative would be to pack all this into a python function and just call it from the PythonOperator

Thanks!

**Comment by artwr** [comment]
This sounds like the perfect use case for the SQL [CheckOperator](https://github.com/airbnb/airflow/blob/master/airflow/operators/check_operator.py). Provided you can write your test like this:

```
SELECT 
IF (DATEDIFF(NOW(), update_time) > 1, FALSE, TRUE)
FROM
INFORMATION_SCHEMA.TABLES
where table_name='yourtable'
```

This operator will fail if any result in a row is False. Does this sounds like it would work?

**Comment by alybel** [comment]
I believe in this case this should work just perfectly, thank you! 

I would still be very interested to find out if there is a way to pull data from a database and then work with it in a downstream PythonOperator. Is this somehow possible?

**Comment by artwr** [comment]
I believe it is through the very good work of @jlowin. Check out https://github.com/airbnb/airflow/pull/260
XCom is designed for cross task communication. Note that this is only for metadata type things, so pulling in a binary flag, or a couple of values should be fine, exporting entire tables would not be advisable.

**Comment by alybel** [comment]
great, thanks! I will try this.

**Comment by alybel** [comment]
Probably I am doing sth horribly wrong, help would still be appreciated

1) I could not find a MySqlCheckOperator so I copied the PrestoCheckOperator and adjusted it to work with MySQL --> this went fine
2) I adjusted the **init**.py file in the operators folder such that it would know the MySqlCheckOperator as well --> this went fine

Now, what happens is this: When I start my DAG, it imports everything correctly but crashes when the MySqlCheckOperator is called. 

This is the error message:
TypeError: get_first() got an unexpected keyword argument 'hql'

To me it looks as if the CheckOperator does not distinguish between sql and hql and therefore sends the hql argument to the get_first() function from MySql that expects sql. 

My Solution: I changed _manually_ in the check_operator.py file 'hql=' into 'sql=' in the get_first() function calls. Positive: It Works! Negative: This is probably not the way it should be done.

Any advice?

Thanks again for your help :)
